### PR TITLE
Blog post on Honey

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

File: content/en/blog/honey-data-collection/index.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+We consider even this processing to be excessive. But it may be possible to barely justify it given that Honey's purpose is to find coupon codes for the products that users look at. But Honey collects *a lot* more data.
+
+Indeed, not only product pages from online shops are logged. Instead, Honey saves any visit to a page whose domain the company has classified as an "online shopping website". But many shopping websites don't just include the actual product pages. They often have a multitude of other content, like blog posts or login pages.  
+And Honey goes even further yet: They log page views even for subsites that are on different subdomains. Thus, the user's browsing habits on countless forums, support pages and other sites are also documented. And for all of these pages, the full URLs is saved, even including the document fragment that may allow reconstructing the precise position of the page the user looked at. The logged URLs and included parameters can also contain sensitive data but in any case they allow Honey to get a detailed view into the users' browsing habits.
```
Comment (mal-tee) [inline content/en/blog/honey-data-collection/index.md:126]: typo: "full URLs is saved"

File: content/en/blog/honey-data-collection/index.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+> –<cite>Honey, <a href="https://www.joinhoney.com/privacy">“Honey Privacy and Security Policy”</a></cite> (highlights by us)
+
+Here, we finally find out that page views on shopping sites are tracked. But they still claim that this only happens on shopping sites.
+
+**Our analysis has shown that Honey's statements are not true.** The actual data collection by them far exceeds both what would be necessary and what they describe on their website. Contrary to what they say, Honey track their users' browsing history and collect their users' personal data on a large scale and this is not restricted to retail sites.
+
+## Our complaints
+
+We are convinced that this data collection by Honey goes too way far. According to the GPDR, every processing of personal data needs to be based on one of six possible legal bases (see Art. 6(1) GDPR). The data subject needs to be informed of the legal basis that is used at the time of the processing (see Art. 13(1)(c) and (d) GDPR). As Honey hasn't done that, Benni explicitly asked them. That's what they replied:
```
Comment (mal-tee) [inline content/en/blog/honey-data-collection/index.md:233]: "too way far" sounds weird to me

**Review by mal-tee — APPROVED** [review]
Thank you very much! LGTM. I left two small notes.

**Comment by baltpeter** [comment]
@mal-tee Fixed. Thanks for the review!

**Pull request merged**